runtime/common/ServerHelper.h

```diff
@@ -14,6 +14,7 @@
 #include "Future.h"
 #include "MeasureCounts.h"
 #include "Registry.h"
+#include "cudaq/orca.h"
```

Comment:
I think the need to add orca-specific dependency at this level (common) is undesirable.

Since we would override the `cudaq::QPU` interface for ORCA anyway, we could encapsulate all changes via inheritance, e.g.,

```
class OrcaServerHelper: public ServerHelper {
public:
// Add a method, not override
ServerJobPayload createJob(cudaq::orca::TBIParameters params);
}

class OrcaExecutor: public Executor {

// Added method, no need to introduce to base clas
details::future execute(cudaq::orca::TBIParameters params,
                          const std::string &kernelName) {

						 ....
auto casted = dynamic_cast<OrcaServerHelper*>(serverHelper);
assert(casted);
auto [jobPostPath, headers, jobs] = casted->createJob(params);						 
						  
				....		  
}

}

class OrcaRemoteRESTQPU : public cudaq::QPU {


  std::unique_ptr<OrcaExecutor> executor; // use OrcaExecutor
	
	
  OrcaRemoteRESTQPU() : QPU() {
    ....
    executor = std::make_unique<OrcaExecutor>();
  }
}
```

At a cost of a `dynamic_cast`, we could encapsulate/localize all ORCA-specific changes. 
@amccaskey interested to hear your opinion as well.

Reply:
I am happy with either solution

Reply:
Addressed in 9b979f2bd2ee0c5ad558c70211416a4aff26f442.